Update SAB article to extend OT until M109 (#3404)
GoogleChrome/developer.chrome.com: site/en/blog/enabling-shared-array-buffer/index.md

  SharedArrayBuffer will arrive in Android Chrome 88. It will only be available
  to pages that are cross-origin isolated. Starting in Desktop Chrome 92 it will
  also only be available to cross-origin isolated pages. You can register for an
  origin trial to retain the current behavior until Desktop Chrome 109.
origin_trial:
  url: /origintrials/#/view_trial/303992974847508481
date: 2021-01-18
updated: 2022-08-08
hero: image/CZmpGM8Eo1dFe0KNhEO9SGO8Ok23/tWnZEOnNmBeFcZxuR9Dx.jpg
alt: A collection of padlocks.
---

{% Aside %}

**Update, Aug 2022**

> To secure more time to reliably relax the requirement to enable cross-origin
> isolation, we've decided to extend the deprecation trial of `SharedArrayBuffer`
> on desktop further until Chrome 109. 

Reviewer: ```suggestion
To secure more time to reliably relax the requirement to enable cross-origin
isolation, the deprecation trial of `SharedArrayBuffer`
on desktop will be extended until Chrome 109. 
```
